docs/src/index.md

```diff
@@ -5,16 +5,99 @@ matrices that have uncertain parameters.
 
```

Comment:
````suggestion
```@meta
CurrentModule = IntervalArithmetic
```

````
I am not sure if this helps because locally I can build the docs just fine.